release_tools/README.adoc

```diff
@@ -0,0 +1,92 @@
+= How to perform a semi-automatic release =
+
+=== Prerequisities ===
+
+* System with `python3.6` or greater, installed https://pypi.python.org/pypi/PyGithub/1.35[PyGithub] module.
+* A GitHub token - get one on https://github.com/settings/tokens. Choose `repo` scope.
+* Your Jenkins user - your User ID should be the same as your GitHub handle, to be sure, check your profile page in Jenkins.
+* A Jenkins token - gnerate a token on Configure page in Jenkins
```

Comment:
I'd say there's no need for indefinite article `A` in list of items (same for "**A** GitHub token").
+ misspelled `generate` (`gnerate`).